Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+    - none: By selecting this option no contrasts are calculated. This option is selected by default. 
+    - deviation: By selecting this contrast the mean of each level of the independent variable is compared with the overall mean (the mean when all the levels are taken together). 
+    - simple: When this contrast is selected the mean of each level is compared to the mean of a specified level, for example with the mean the control group. 
+    - difference: This contrast is also called reverse Helmert. By selecting this contrast the mean of each level is compared to the mean of the previous levels. 
+    - Helmert: When this contrast is selected the mean of each level is compared to the mean of the following levels. This is the reverse of the difference contrast. 
+    - repeated: By selecting this contrast the mean of each level is compared to the mean of the following level. 
+    - polynomial: This contrast test polynomial trends in the data. The specific polynomial that will be used for the analysis depends on the amount of levels of the independent variable. The degree of the trend used for the analysis is the amount of levels - 1. Therefore, if the independent variable consist of 2 levels, a linear trend is analysed. If the independent variable consists of three levels, a quadratic trend is analysed besides to linear. 
+- Assume equal variances: You can select this option when you assume that the variances of the levels of the independent variable are similar. This option is selected by default. 
+- Confidence Intervals: By selecting this option, confidence intervals for the estimated mean difference will be given. By default this is set to 95%. You can change this into the desired percentage. 
+
+#### Post Hoc Tests: 
+If there is a significant effect of the independent variable when is accounted for the covariate, you can follow up the analysis by performing a post hoc test to see which levels of a independent variable exactly differ. To perform a post hoc test drop the factor to perform the post hoc test on in the right column. Then you can select:    
+- Effect size: By selecting this option the effect size, referred as the magnitude of the observed effect, will be presented. The used measure for the effect size is Cohen's d. The effect size will only be presented for the post hoc type `Standard`. 
+- Confidence intervals: When this option is selected, the confidence interval for the mean difference is calculated. This is done for every post hoc method except for Dunn. By default this is set to 95% but you can adjust this into the desired percentage.  
+- Correction: To correct for multiple comparison testing and avoid Type I errors, you can select different correction methods. 
+    - Tukey: Compare all possible pairs of group means. This correction can be used when the groups of the independent variable have the same sample size and variance. It is a considered a common used correction and is selected by default. 
```

Comment:
have an equal sample size and variance. This method is commonly used and is selected by default.